### DISCUSSION TITLE
Class Action indicator?

### DISCUSSION THREAD

**Discussion by EagleWorks** [freelawproject/courtlistener#4770] [Q&A] [answered]
On docket objects, the value of the `jury_demand` key indicates which party, if any, has demanded a jury trial.

Is there an equivalent indicator for whether a case is proposed as a _Class Action_ suit? I did not see one in the `search_docket` object, and no indicator of such on CL's rendered HTML.

( My use is low traffic enough that I can parse this out of the complaint filings manually after using the API to populate most other data. This discussion is primarily to scratch a curiosity itch. )

**Comment by mlissner** [comment, marked as answer]
I don't think we can get class action status from any particular field, but it's something we could probably get from filings. Haven't done so though...
